google-cloud-notification/pom.xml

```diff
@@ -6,7 +6,7 @@
   <artifactId>google-cloud-notification</artifactId>
   <version>0.45.1-beta-SNAPSHOT</version><!-- {x-version-update:google-cloud-notification:current} -->
   <packaging>jar</packaging>
-  <name>Google Cloud Pub/Sub Notifications for Google Cloud Storage</name>
+  <name>Google Cloud Pub/Sub Notifications for GCS</name>
```

Comment:
Why this abbreviation?

Reply:
Too long string makes maven output look ugly